ocs_ci/ocs/node.py

```diff
@@ -359,3 +359,22 @@ def get_node_resource_utilization_from_oc_describe(nodename=None, node_type='wor
         }
 
     return utilization_dict
+
+
+def get_node_from_machine_name(machine_name):
```

Comment:
Instead, you can get the machine obj and get the node name using it  as below,

machine_objs = get_machine_objs()
    for machine in machine_objs:
        if machine.get().get(
                'status'
        ).get('addresses')[1].get('address')

Reply:
@prsurve This comment is not addressed. Any particular reason?

Reply:
i have made changes local i need to test and will update shortly

Reply:
Done